Rubberduck.VBEditor.VB6/SafeComWrappers/VB/Windows.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections;
+using System.Collections.Generic;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using VB = Microsoft.Vbe.Interop.VB6;
+
+// ReSharper disable once CheckNamespace - Special dispensation due to conflicting file vs namespace priorities
+namespace Rubberduck.VBEditor.SafeComWrappers.VB6
+{
+    public class Windows : SafeComWrapper<VB.Windows>, IWindows
+    {
+        public Windows(VB.Windows target, bool rewrapping = false)
+            : base(target, rewrapping)
+        {
+        }
+
+        public int Count => IsWrappingNullReference ? 0 : Target.Count;
+
+        public IVBE VBE => new VBE(IsWrappingNullReference ? null : Target.VBE);
+
+        public IApplication Parent => new Application(null); // VB6 has no Application
+
+        public IWindow this[object index] => new Window(IsWrappingNullReference ? null : Target.Item(index));
+
+
+        private static readonly Dictionary<VB.Window, object> _dockableHosts = new Dictionary<VB.Window, object>();
```

Comment:
As a note for further PRs, we should see whether we can make that a dictionary over `IWindow`, maybe using the `ReferenceEqualityComparer` to ignore any custom implementations of `GetHashCode` and `Equals`.
The same applies to the VBA version.

Reply:
#3957